extension/test/e2e/credit-card-3ds-redirect.spec.js

```diff
@@ -0,0 +1,146 @@
+const nodeStatic = require('node-static')
```

Comment:
why not simply using my implementation instead of changing it, I spend time and explained better the tricky redirect part 😉